src/args.rs

```diff
@@ -170,6 +174,39 @@ fn parse_auth(src: &str) -> Result<auth::RequiredAuth, ContextualError> {
     })
 }
 
+/// A own header modified from [httparse](https://docs.rs/httparse/1.3.5/src/httparse/lib.rs.html#415-425)
+#[derive(Clone, Eq, PartialEq, Debug)]
+pub struct Header {
```

Comment:
Hm, do we really need this custom struct? It's not a problem if we really do but I'd like to understand the reason first.